src/main/java/carracing/domain/CarFactory.java

```diff
@@ -1,18 +1,17 @@
 package carracing.domain;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 public class CarFactory {
 
     private CarFactory() {
     }
 
-    public static List<Car> makeCars(int count) {
-        List<Car> cars = new ArrayList<>(count);
-        for (int i = 0; i < count; i++) {
-            cars.add(new Car());
-        }
+    public static List<Car> makeCars(String[] names) {
+        List<Car> cars = new ArrayList<>(names.length);
+        Arrays.stream(names).forEach(name -> cars.add(new Car(name)));
         return cars;
```

Comment:
```suggestion
        return Arrays.stream(names)
                        .map(Car::new)
                        .collect(toList());
```

와 같이 사용할 수 있을 것 같습니다!
stream 마무리 연산자에 대해 공부해보아요 :)